### PR TITLE
feat(metrics): Add metrics about `disableAnonymity` in retro meetings

### LINKED ISSUE #7163: Include `disableAnonymity` in Meeting objects when sending metrics to Segment
## Issue - Enhancement

We released a new feature to disable anonymity in retro meetings. We'd like to know how many retro meetings have this feature turned on.

### Acceptance Criteria (optional)
- [ ] On `Meeting Started`, `Meeting Joined` and `Meeting Completed` events, include the boolean property `disableAnonymity`
- [ ] When the facilitator toggles the `disableAnonymity` setting, emit an event

**Estimated effort:** X points ([see CONTRIBUTING.md](https://github.com/ParabolInc/parabol/blob/master/CONTRIBUTING.md#points-and-sizes))

### PR DESCRIPTION
**Pull request by tianrunhe**
# Description

Fixes #7163 

## Demo
New property `disableAnonymity` included in Meeting related events:
Screen Shot 2022-09-12 at 3 43 30 PM

New event `Meeting Settings Changed` published to Segment:
Screen Shot 2022-09-12 at 3 42 39 PM

## Testing scenarios

- [ ] New property `disableAnonymity` included in Meeting related events
  - Start/Join/Complete a meeting
  - Verify `disableAnonymity` is included in the relevant events in Segment

- [ ] New event `Meeting Settings Changed` published to Segment:
  - Go to meeting selector modal
  - Toggle the `Anonymous reflections` setting
  - Verify a new `Meeting Settings Changed` event was published and the `disableAnonymity` property is expected

## Final checklist

- [x] I checked the [code review guidelines](../docs/codeReview.md)
- [x] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [x] I have performed a self-review of my code, the same way I'd do it for any other team member
- [x] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [x] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [x] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [x] PR title is human readable and could be used in changelog

### CODE REVIEW

**Review by igorlesnenko — CHANGES_REQUESTED** [review]
- Open retro settings 
- remove checkbox from anonymous reflections, see in segment logs:
```
disableAnonymity: true,
    hasIcebreaker: false,
```

I guess `hasIcebreaker` should be true?

**Review by igorlesnenko — APPROVED** [review]

File: packages/server/graphql/mutations/setMeetingSettings.ts [hunk 1]
```diff
@@ -44,8 +45,9 @@ const setMeetingSettings = {
     if (!settings) {
       return standardError(new Error('Settings not found'), {userId: viewerId})
     }
-    const {teamId} = settings
+    const {teamId, meetingType} = settings
 
+    const meetingSettings = {}
```
Comment (Dschoordsch) [inline packages/server/graphql/mutations/setMeetingSettings.ts:50]: -1 You can type this here `const meetingSettings = {} as MeetingSettings` then you don't need to do the `Object.assign` trick to bypass typings.

File: packages/server/graphql/mutations/setMeetingSettings.ts [hunk 2]
```diff
@@ -58,17 +60,20 @@ const setMeetingSettings = {
             checkinEnabled ? row('phaseTypes') : row('phaseTypes').difference(['checkin']),
             checkinEnabled ? row('phaseTypes').prepend('checkin') : row('phaseTypes')
           )
+          Object.assign(meetingSettings, {hasIcebreaker: checkinEnabled})
```
Comment (Dschoordsch) [inline packages/server/graphql/mutations/setMeetingSettings.ts:63]: -1 If you type it properly, you can just do `meetingSettings.hasIcebreaker = checkinEnabled` here which is more readable.

File: packages/server/graphql/mutations/setMeetingSettings.ts [hunk 3]
```diff
@@ -58,17 +60,20 @@ const setMeetingSettings = {
...
         }
 
         if (isNotNull(disableAnonymity)) {
           updatedSettings.disableAnonymity = disableAnonymity
+          Object.assign(meetingSettings, {disableAnonymity})
```
Comment (Dschoordsch) [inline packages/server/graphql/mutations/setMeetingSettings.ts:68]: -1 `meetingSettings.disableAnonymity = disableAnonymity`

**Review by Dschoordsch — CHANGES_REQUESTED** [review]

**Review by Dschoordsch — APPROVED** [review]

**Pull request merged**